src/vivarium_public_health/risks/effect.py

```diff
@@ -77,9 +78,12 @@ def setup(self, builder):
 
         self.exposure_effect = get_exposure_effect(builder, self.risk)
 
-        builder.value.register_value_modifier(f'{self.target.name}.{self.target.measure}', modifier=self.adjust_target)
+        builder.value.register_value_modifier(f'{self.target.name}.{self.target.measure}',
+                                              modifier=self.adjust_target,
+                                              requires_values=[f'{self.risk.name}.exposure'])
```

Comment:
I think this also needs `requires_columns=['age', 'sex']` because it uses the rr lookup table

Reply:
Yeah, I now don't like how hidden all that is.